presto-spi/src/main/java/com/facebook/presto/spi/block/Block.java

```diff
@@ -167,6 +167,11 @@ default int compareTo(int leftPosition, int leftOffset, int leftLength, Block ri
      */
     long getSizeInBytes();
 
+    /**
+     * Returns the content size of a block if no optimizations were applied to it (e.g RLE, Dictionary).
+     */
+    long getUnoptimizedSizeInBytes();
```

Comment:
If the implementation is the same for most subclasses, it may be simpler to just add a default implementation here.